mods/ts/mod.yaml

```diff
@@ -270,12 +271,21 @@ ModContent:
 			Download: basefiles
 		music: Base Game Music
 			TestFiles: ^Content/ts/scores.mix
-			Sources: tibsun, tibsun-linux, tfd, origin
+			Sources: tibsun, tibsun-linux, tfd, origin, firestorm
 			Download: music
+		expansion: Firestorm Expansion
+			TestFiles: ^Content/ts/expand01.mix
+			Sources: firestorm, tfd
+			Required: true
```

Comment:
For a quick install I guess we need to know what we actually want for a stripped down redundancy removed archive. I know too little about the inner workings of those files, so I will keep my hands off, but I know many people who have been working with these files way longer than I have do. I still hope they will one day contribute to this repository themselves.